### PR TITLE
Add evaluation of K

### PR DESCRIPTION
**Pull request by SvenVw**

### CODE REVIEW

File: R/potassium.R [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  # Load in the datasets
+  crops.obic <- as.data.table(OBIC::crops.obic)
+  setkey(crops.obic, crop_code)
+  soils.obic <- as.data.table(OBIC::soils.obic)
+  setkey(soils.obic, soiltype)
+  
+  # Check inputs
+  arg.length <- max(length(A_PH_CC), length(A_OS_GV), length(A_CEC_CO), length(A_K_CEC), 
+                    length(A_K_CC), length(A_CLAY_MI), length(B_BT_AK), length(B_LU_BRP))
+  checkmate::assert_numeric(A_PH_CC, lower = 3, upper = 10, any.missing = FALSE, len = arg.length)
```
Comment (JobdePater) [inline R/potassium.R]: waarden deze checks in help geven. dan weet de gebruiker wat de mogelijke ranges zijn

File: R/potassium.R [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  setorder(dt, id)
+  value <- dt[, value]
+  
+  # return value: be aware, index is different for different land use and soil combinations
+  return(value)
+  }
+
+#' Calculate the indicator for Potassium Availability
+#' 
+#' This function calculates the indicator for the the Potassium Availability of the soil by using the Mg-availability calculated by \code{\link{calc_potassium_availability}}
```
Comment (JobdePater) [inline R/potassium.R]: Mg-availability => K-availability

File: R/potassium.R [hunk 3]
```diff
@@ -0,0 +1,195 @@
+#' Calculate the K availability
+#' 
+#' This function calculates the K availability of a soil.
+#' 
+#' @param A_PH_CC (numeric) The acidity of the soil, measured in 0.01M CaCl2 (-)
+#' @param A_OS_GV (numeric) The organic matter content of the soil (in procent)
```
[the remark this answers is not in the document]
Comment (gerardhros) [reply R/potassium.R]: unit checking for uniform format is indeed needed for consistency. will be done in other branch.

File: R/potassium.R [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+#' @param A_CEC_CO (numeric) The cation exchange capacity of the soil (mmol+ per kg), analysed via Cobalt-hexamine extraction
+#' @param A_K_CC (numeric) The plant available potassium, extracted with 0.01M CaCl2 (mg per kg), 
+#' @param A_K_CEC (numeric) The occupation of the CEC with potassium (in procent)
+#' @param A_CLAY_MI (numeric) The clay content of the soil (in procent)
+#' @param B_BT_AK (character) The type of soil
+#' @param B_LU_BRP (numeric) The crop code (gewascode) from the BRP
```
[the remark this answers is not in the document]
Comment (gerardhros) [reply R/potassium.R]: technical details over input ranges (upper and lower) and options will be given in OBIC vignette

File: R/potassium.R [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+#' 
+#' @import data.table
+#' 
+#' @export
+calc_potassium_availability <- function(A_K_CC, A_K_CEC,A_CEC_CO, A_PH_CC, A_OS_GV, A_CLAY_MI,
```
[the remark this answers is not in the document]
Comment (gerardhros) [reply R/potassium.R]: correct. will be done in a next version.

**Pull request merged**